virtual_workstation_setup.sh

```diff
@@ -116,6 +116,8 @@ idem_file_append() {
 # session.
 setup_bash() {
     # Bash profile should source .bashrc
+    echo "################################################################################"
```

Comment:
I added some visual indicators to each step of the script so it was easier to follow along with what is was doing as I was running it for testing.